python_modules/dagster/dagster/_core/definitions/multi_dimensional_partitions.py

```diff
@@ -334,6 +337,111 @@ def get_partition_keys(
             current_time or get_current_datetime(), dynamic_partitions_store
         )
 
+    def get_partition_key_connection(
+        self,
+        current_time: Optional[datetime] = None,
+        dynamic_partitions_store: Optional[DynamicPartitionsStore] = None,
+        cursor: Optional[str] = None,
+        limit: Optional[int] = None,
```

Comment:
should we require a limit?

Reply:
Yeah, I flipped back and forth about this.  Probably makes sense to do this.